### ISSUE TITLE
feat: Add `no-console` rule to `deno lint`

### ISSUE THREAD

**Issue by marvinhagemeister** [denoland/deno#20187]
I'm guilty of accidentally committing `console.log` statements from time to time. We should have a `no-console` rule, similar to eslint: https://eslint.org/docs/latest/rules/no-console

**Comment by dsherret** [comment]
Me too https://github.com/dsherret/dax/pull/163

**Comment by 0xIchigo** [comment]
@marvinhagemeister @dsherret @bartlomieju I've opened up a PR in [deno_lint](https://github.com/denoland/deno_lint) which would allow for the implementation of a "no console log" rule. Once that is approved, I'd be more than happy to add this rule to `.dlint.json` to implement the rule in the main Deno repository 🔥

**Comment by scarf005** [comment]
despite https://github.com/denoland/deno_lint/pull/1189 being merged and released in `v0.51.0`, and deno CLI using [deno_lint `v0.52.0`](https://github.com/denoland/deno/blob/585cf2de89f1d253bc14e0835790445d14e324f3/cli/Cargo.toml#L59), I was unable to use this rule. Could there be a step I've been missing?

```
 ~/r/e/no-console  deno lint --rules | grep no-console
 !  ~/r/e/no-console  echo $status
```
```jsonc
{
    "lint": {
        "rules": { "include": ["no-console"] }
    }
}
```

**Comment by dsherret** [comment]
Looks like it wasn't properly hooked up. I just opened https://github.com/denoland/deno_lint/pull/1214
